spec/lib/myfinance/resources/receivable_account_spec.rb

```diff
@@ -82,4 +84,30 @@
       end
     end
   end
+
+  describe "#receive", vcr: true do
+    let(:params) { { total_amount: 150.99, occurred_at: '2015-08-05', amount: 150.99 } }
+    subject { client.receivable_accounts.receive(1215634, entity_id, params) }
+
+    context "with success" do
+      it "receives the receivable account" do
+        expect(subject.id).to eq(1215634)
+        expect(subject.status).to eq(2)
+        expect(subject.status_name).to eq('received')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.